app/models/comment.rb

```diff
@@ -36,6 +36,8 @@ def check_for_spam
   scope :ordered_by_date, -> { order('created_at DESC') }
   scope :top_level,       -> { where.not(commentable_type: "Comment") }
   scope :include_pseud,   -> { includes(:pseud) }
+  scope :include_user,    -> { includes(pseud: [:user]) }
+  scope :include_parent,  -> { includes(:parent) }
```

Comment:
Since the changes to the comments controller were removed, these are now unused and should be removed as well.